go.mod

```diff
@@ -1,29 +1,29 @@
 module sigs.k8s.io/descheduler
 
-go 1.18
+go 1.19
 
 require (
 	github.com/client9/misspell v0.3.4
 	github.com/spf13/cobra v1.4.0
 	github.com/spf13/pflag v1.0.5
-	k8s.io/api v0.24.0
-	k8s.io/apimachinery v0.24.0
-	k8s.io/apiserver v0.24.0
-	k8s.io/client-go v0.24.0
-	k8s.io/code-generator v0.24.0
-	k8s.io/component-base v0.24.0
-	k8s.io/component-helpers v0.24.0
-	k8s.io/klog/v2 v2.60.1
-	k8s.io/kubectl v0.20.5
-	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
+	k8s.io/api v0.25.0
+	k8s.io/apimachinery v0.25.0
+	k8s.io/apiserver v0.25.0
+	k8s.io/client-go v0.25.0
+	k8s.io/code-generator v0.25.0
+	k8s.io/component-base v0.25.0
+	k8s.io/component-helpers v0.25.0
+	k8s.io/klog/v2 v2.70.1
+	k8s.io/kubectl v0.25.0
```

Comment:
should we merge https://github.com/kubernetes-sigs/descheduler/pull/938 first, or after this?

Reply:
looks like #938 depends on this (specifically the go1.19 bump), so we can merge this first